test/test_load_old_agent.py

```diff
@@ -23,6 +23,7 @@
 from oneagent.common import AgentState
 
 @pytest.mark.dependsnative
+@pytest.mark.skipif('DT_OLDAGENTLIBRARY' not in os.environ, reason="See OA-8673, EP-918")
```

Comment:
Should we maybe exchange the ticket numbers with a small description?

Reply:
I guess these aren't really useful for non-Dynatracers, right?
Reason is that we no longer keep easily downloadable binaries of very old agent versions around and the test just was not very helpful.

Reply:
Done in 555009d